### PR TITLE
Updated pytest.yml

### LINKED ISSUE #31: Change github workflow to run on pull requests
Change this line 

https://github.com/arshadkazmi42/currency-symbols/blob/master/.github/workflows/pytest.yml#L3 

similar to these lines

https://github.com/gh-conf/rhof/blob/master/.github/workflows/nodejs.yml#L3-L5

### PR DESCRIPTION
**Pull request by Amank-root**
Fixes #31

### CODE REVIEW

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Python CI
 
-on: [push]
+on:
```
Comment (arshadkazmi42) [inline .github/workflows/pytest.yml:3]: We need to add both push and pull request

```
on:
  - push
  - pull_request
```

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

There one small change. Once that is fixed, we can get this merged.

File: .github/workflows/pytest.yml (hunk 1 above)
Comment (Amank-root) [reply .github/workflows/pytest.yml:3]: Okay I'll do that

**Review by arshadkazmi42 — APPROVED** [review]
LGTM :tada:

**Pull request merged**